### DISCUSSION TITLE
Is there any way to view the documentation of previous neovim releases online?

### DISCUSSION THREAD

**Discussion by Eutropios** [neovim/neovim#29275] [Q&A] [answered]
Is there any way to view the documentation of previous neovim releases on the website?

**Comment by clason** [comment]
No.

**Reply by Eutropios** [reply]
Is there instead a way to view it locally? Would I have to install said version to view its documentation?

**Reply by clason** [reply, marked as answer]
Yes. 

(Sorry for the one-word answers, but there really is nothing else to say.)

**Reply by Eutropios** [reply]
Lol, no worries. I appreciate your help!
